packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -16,18 +16,22 @@
  * public/private key pairs. If you're publishing your code to GitHub make sure you load this
  * phrase from a file you've .gitignored so it doesn't accidentally become public.
  *
+ * It is recommended to store your secret variables in a .env file (which you should include in .gitignore).
+ * In your project root, run `$ npm install dotenv`.
+ * Create .env and declare your MNEMONIC and infura PROJECT_ID variables inside.
+ * .gitignore is already added with this project with .env in it.
  */
 
+// require('dotenv').config();
 // const HDWalletProvider = require('@truffle/hdwallet-provider');
-//
-// const fs = require('fs');
-// const mnemonic = fs.readFileSync(".secret").toString().trim();
+
+// const { MNEMONIC, PROJECT_ID } = process.env;
```

Comment:
Maybe the require dotenv line should be adjacent to the destructure line, and separated from the hdwallet-provider line? Like so:

```
require dotenv
destructure

hdwallet-provider
```

Reply:
Yes, we can do this to make `dotenv` usage clearer. I will change this. Thanks for pointing that out @cliffoo.